src/d3a/models/strategy/area_agents/two_sided_pay_as_clear_engine.py

```diff
@@ -107,7 +110,7 @@ def _match_offers_bids(self):
             return
         clearing_rate, clearing_energy = clearing
         if clearing_energy > 0:
-            self.owner.log.info(f"Market Clearing Rate: {clearing_rate} "
+            self.owner.log.warn(f"Market Clearing Rate: {clearing_rate} "
```

Comment:
I guess you can revert this again, right?

Reply:
IMO looks better with warn, to see it with trade details

Reply:
OK

Reply:
I think I added this comment on another PR yesterday, but I do not think it is that important, since you can see the market clearing rate directly from the trades. I don't have a strong opinion though, since it probably will not clatter the logs that much.
In addition to that, if you decide to stick to having the log, I would suggest to use the log.warning method instead of the deprecated log.warn.